Xamarin.Forms.Core/Label.cs

```diff
@@ -42,35 +42,32 @@ public class Label : View, IFontElement, ITextElement, ITextAlignmentElement, IL
 				if (oldvalue != null)
 				{
 					var formattedString = ((FormattedString)oldvalue);
+					var label = ((Label)bindable);
 
-					// Remove spans here, to ensure all collection watching, removes necessary event hooks.
-					for (int i = formattedString.Spans.Count - 1; i >= 0; i--)
-						formattedString.Spans.RemoveAt(i);
-
```

Comment:
This seemed like an unnecessary breaking change.  This causes our code to just remove all the elements from the Span without user input. So if the user has a FormattingString they are storing locally then this creates the side effect of clearing out the span. 

The UI Test here demonstrates that idea
https://github.com/xamarin/Xamarin.Forms/pull/3643/files#diff-cb5bd463d76b7b289342ccdf445e96b7R142